### ISSUE TITLE
Checkbox to Toggle UI and copy consistency improvements

### ISSUE THREAD

**Issue by dstrukt** [btcpayserver/btcpayserver#5734]
In lieu of the update to the `Policies` tab and a change to some of the settings screens, I've been auditing all views and picking out the places we use checkboxes, when we likely should be using toggles to improve the functional legibility, and also the inconsistencies in copy / functionality (e.g. You enable a checkbox to "disable" it, it's inverted logically speaking, see first item)

# Store

## Store > Email Server

- [x] `Disable TLS certificate security checks` -> `TLS certificate security checks` & change to toggle

<img width="1026" alt="Screen Shot 2024-02-05 at 10 59 27 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/314425d9-2f10-45a6-b708-810ab81fcde1">

## Store > Payout Processors

- [x] change to toggle in on-chain 
- [x] change to toggle in off-chain

<img width="422" alt="Screen Shot 2024-02-05 at 11 04 15 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/47d097c0-37ac-4391-b8da-83e73dafa751">
<img width="411" alt="Screen Shot 2024-02-05 at 11 04 22 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/13d66eef-590c-494d-9197-922d1992b670">

## Store > Checkout & Receipt

- [x] all of these should be toggles
<img width="952" alt="Screen Shot 2024-02-05 at 11 16 01 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/806c3e55-d1e9-47fc-8112-d8754a13455b">

- [ ] all of these should be toggles
<img width="498" alt="Screen Shot 2024-02-05 at 11 16 04 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/7bddd6f7-6fba-4d96-98ec-edfdc4dadfce">

# Server Settings

[TODO]

# Account Settings

## Account > Notifications

- [x] all toggles
- [ ] out of scope, but functionally the `Disable All` once clicked should switch to `Enable All` and it just enables/disables all and you have to hit `Save` for the state change to take place, currently it's not as useable as it could be .. might make a separate ticket
<img width="921" alt="Screen Shot 2024-02-05 at 11 30 51 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/aeb85f63-7818-4031-a8e5-1406db2bc843">

# Crowdfund Settings

## Crowdfund 
- [x] should all be toggles
<img width="610" alt="Screen Shot 2024-02-05 at 11 25 38 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/72c6ed9c-87ac-4893-9a6d-94f33526c24c">

## Item Editor Setting
- [ ] `Disabled` -> `Disable` OR we change to `Enable` and have it be toggled ON, so you have to turn it off to disable
- [ ] Whatever we do the above should be done to the Point of Sale setting as well
<img width="364" alt="Screen Shot 2024-02-05 at 11 25 41 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/277f141e-f015-40eb-a217-e5300719b4d6">

# Bitcoin Wallet

- [ ] should be a toggle

<img width="550" alt="Screen Shot 2024-02-05 at 11 22 05 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/e34fd851-45aa-4fd1-9cad-e63ada52a813">

# Lightning Node

[TODO]

# Payment Requests > Create

- [ ] should be a toggle

<img width="963" alt="Screen Shot 2024-02-05 at 11 37 51 PM" src="https://github.com/btcpayserver/btcpayserver/assets/6250771/070c0120-57da-4dbc-83cb-68b78fa04d4a">

**Comment by dennisreimann** [comment]
Yes, we can tackle this once #5731 is done.

**Comment by dstrukt** [comment]
Sounds good - no rush, just minor tweaks to improve our various setting views!

**Comment by TChukwuleta** [comment]
Can I hop on this?

**Comment by dennisreimann** [comment]
Please wait until #5731 is done.

**Comment by TChukwuleta** [comment]
Sure.

**Comment by pavlenex** [comment]
You can tackle it now @TChukwuleta
